pkg/cincinnati/cincinnati.go

```diff
@@ -90,7 +90,7 @@ func (c Client) GetUpdates(upstream string, channel string, version semver.Versi
 		}
 	}
 	if !found {
-		return nil, fmt.Errorf("unknown version %s", version)
+		return nil, fmt.Errorf("version %s is not in the %s channel", version, channel)
```

Comment:
I'm not sure what the best wording here is, though it should highlight that it's talking about the currently installed version which is not found in the updates available for the requested channel. May be "`currently installed version %q does not exist in channel %q`?

Reply:
> May be "`currently installed version %q does not exist in channel %q`?

I've pushed 310884a -> 3dca6ac, rebasing on master and adjusting to:

    current version %s not found in the %q channel

Differences vs. your suggestion:

* "currently installed" -> "current".  I think the single word is clear enough.
* `%q` -> `%s` for the version.  We are requiring a `semver.Version` for `version`, so I don't see an opportunity for an empty or whitespace string there.  Quoting the channel makes sense, so I've followed you there.
* "does not exist" -> "not found", to be a bit less opinionated about why we haven't found our version.

Thoughts?

Reply:
> "currently installed" -> "current". I think the single word is clear enough.

in case we're not trying to save bytes or screenspace I encourage to add the additional word. A user might otherwise misinterpret the error, e.g. thinking that they requested a specific version to be upgraded to and getting confused about it. 

>     * `%q` -> `%s` for the version.  We are requiring a `semver.Version` for `version`, so I don't see an opportunity for an empty or whitespace string there.  Quoting the channel makes sense, so I've followed you there.

:+1: (though I generally prefer single-quotes around variable values for readability; didn't think of escaping)

> "does not exist" -> "not found", to be a bit less opinionated about why we haven't found our version.

:+1:

Reply:
> in case we're not trying to save bytes or screenspace I encourage to add the additional word. A user might otherwise misinterpret the error, e.g. thinking that they requested a specific version to be upgraded to and getting confused about it.

I think "current version" is clearly not "the version you want to upgrade to", but whatever, I've updated to "currently installed version" with 3dca6ac -> cb4d7b1.